pkg/volume/csi/csi_attacher_test.go

```diff
@@ -221,6 +221,8 @@ func TestAttacherAttach(t *testing.T) {
 				status.AttachError = &storage.VolumeError{
 					Message: "attacher error",
 				}
+				errStatus := apierrs.NewInternalError(fmt.Errorf("we got an error")).Status()
+				fakeWatcher.Error(&errStatus)
```

Comment:
Would it be better to send the entire error instead of just the status ?

Reply:
Given
```
				statErr := apierrs.NewInternalError(fmt.Errorf("we got an error"))
```
when I send the above error, I get compilation error saying that funcs such as GetObjectKind are missing.

Do you have something else in mind, @vladimirvivien ?

Reply:
Ok my apologies, `fakewatcher.Error(..)` takes an api object not an error.  The code is ok as is.